libs/utils/CMakeLists.txt

```diff
@@ -39,12 +41,15 @@ add_library(utils SHARED
     src/ip_utils.c
     src/filter.c
     src/celix_log_utils.c
+    src/celix_file_utils.c
     ${MEMSTREAM_SOURCES}
 )
 set_target_properties(utils PROPERTIES OUTPUT_NAME "celix_utils")
+target_link_libraries(utils PRIVATE libzip::libzip)
 
 if (NOT OPEN_MEMSTREAM_EXISTS)
     target_compile_definitions(utils PUBLIC -DNO_MEMSTREAM_AVAILABLE)
+    target_compile_definitions(utils PUBLIC -DNO_MEMSTREAM_AVAILABLE)
```

Comment:
Unintentional added copy of the previous line? (at least, I hope it is unintentional).

Reply:
double line removed